www/docs/content/recipes/commerce-automation.mdx

```diff
@@ -0,0 +1,655 @@
+
+By utilizing the event bus service, you can handle events within an order workflow to automate actions. For example, when an order is placed, you can listen to the `order.placed` event and automatically create a fulfillment if a set of predefined conditions are met. Another example is automatically capturing payment when an order is fully shipped and the `order.shipment_created` is triggered.
+
+Medusa’s commerce features are also geared towards automating RMA flows and ensuring a good customer experience.
+
+For example, customers can create returns for their orders without direct involvement from the admin. The admin will then receive a notification regarding the return and handle it accordingly. The same applies to order exchanges.
+
+Medusa also allows admins to edit orders, receive customer approval for the edits, and authorize additional payment if necessary, all within a seamless workflow.
+
+Similar to the examples mentioned earlier, each of these actions trigger events that you can listen to and perform additional actions based on your use case.
+
+<DocCardList colSize={4} items={[
+  {
+    type: 'link',
+    href: '/development/events',
+    label: 'Event Bus',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Learn about the event architecture system in the Medusa backend.',
+    }
+  },
+  {
+    type: 'link',
+    href: '/development/events/create-subscriber',
+    label: 'Create a Subscriber',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Learn how to create a subscriber to listen to events.',
+    }
+  },
+  {
+    type: 'link',
+    href: '/development/events/events-list',
+    label: 'Events Reference',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Check out triggered events in Medusa and their expected payloads.',
+    }
+  },
+  {
+    type: 'link',
+    href: '/modules/orders/returns',
+    label: 'Order Returns',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Learn about the Order Returns architecture and features.',
+    }
+  },
+  {
+    type: 'link',
+    href: '/modules/orders/swaps',
+    label: 'Order Swaps',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Learn about the Order Swaps architecture and features.',
+    }
+  },
+  {
+    type: 'link',
+    href: '/modules/orders/#order-edits',
+    label: 'Order Edits',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Learn about the Order Edits architecture and features.',
+    }
+  },
+]} />
+
+---
+
+## Customer Segmentation
+
+Customer segmentation allows you to organize customers into groups, and then apply different price rules to these groups. For example, you may group customers by their product preferences, the number of orders they’ve placed, or geographical location.
```

Comment:
Something feels weird about the phrase "Customer segmentation allows you to ..."

Would maybe explain why you would want to do customer segmentation in the first place instead